app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2 class="govuk-heading-m"><%= t('.page_heading') %></h1>
```

Comment:
HTML Headings: opening tag is an `h2` but closing tag is an `h1`, which should it be?

Reply:
have changed both to h2